src/api/app/models/token/workflow.rb

```diff
@@ -12,7 +12,7 @@ def call(options)
     scm_extractor_payload = extractor.call
     yaml_downloader = Workflows::YAMLDownloader.new(scm_extractor_payload)
     yaml_file = yaml_downloader.call
-    raise Token::Errors::NonExistentWorkflowsFile, yaml_downloader.errors.join('\n') if yaml_file.blank?
+    raise Token::Errors::NonExistentWorkflowsFile, "issue with workflows.yml: #{yaml_downloader.errors.join('\n')}" if yaml_file.blank?
```

Comment:
We should let users know that we're looking for their `.obs/workflows.yml` file on their SCM's default branch.

Reply:
ideas?